test/index.js

```diff
@@ -385,6 +385,14 @@ test('ChatList methods', t => {
   t.end()
 })
 
+test('key transfer methods', t => {
+  dc.initiateKeyTransfer((err, setupCode) => {
+    t.ok(err == null, 'err is not null')
+    t.ok(setupCode != null, 'setupCode is null')
```

Comment:
Also maybe do `t.is(typeof setupCode, 'string')`. Currently anything _but_ null is ok.